staff/judith-gongora/cv/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <title>Cv</title>
+</head>
+<body>
+        <main>
+            <article>
```

Comment:
articles do not seem to match here in this content, but sections seem to be ok, please RTFM
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section